build.sbt

```diff
@@ -733,9 +733,11 @@ lazy val flinkTests = (project in flink("tests"))
     name := "nussknacker-flink-tests",
     libraryDependencies ++= {
       Seq(
-        "org.apache.flink" % "flink-connector-base"       % flinkV % Provided,
-        "org.apache.flink" % "flink-streaming-java"       % flinkV % Provided,
-        "org.apache.flink" % "flink-statebackend-rocksdb" % flinkV % Provided
+        "org.apache.flink" % "flink-connector-base"       % flinkV               % Provided,
+        "org.apache.flink" % "flink-streaming-java"       % flinkV               % Provided,
+        "org.apache.flink" % "flink-statebackend-rocksdb" % flinkV               % Provided,
+        "org.apache.flink" % "flink-connector-kafka"      % flinkConnectorKafkaV % Provided,
```

Comment:
Shouldn't be in test scope as well?

Reply:
changed all these deps scope to test since its just a test module